packages/atomic/src/components/common/facets/category-facet/search-value.ts

```diff
@@ -0,0 +1,87 @@
+import '@/src/components/common/atomic-icon/atomic-icon';
+import type {FunctionalComponent} from '@/src/utils/functional-component-utils';
+import type {i18n} from 'i18next';
+import {html} from 'lit';
+import {hierarchicalPath} from '../../../../directives/hierarchical-path';
+import {getFieldValueCaption} from '../../../../utils/field-utils';
+import {renderButton} from '../../button';
+import {renderFacetValueLabelHighlight} from '../facet-value-label-highlight/facet-value-label-highlight';
+
+interface CategoryFacetSearchValueProps {
+  value: {count: number; path: string[]; displayValue: string};
+  i18n: i18n;
+  field: string;
+  onClick(): void;
+  searchQuery: string;
+}
+
+const SEPARATOR = '/';
+const PATH_MAX_LENGTH = 3;
+
+export const renderCategoryFacetSearchValue: FunctionalComponent<
+  CategoryFacetSearchValueProps
+> = ({props}) => {
+  const count = props.value.count.toLocaleString(props.i18n.language);
+  const inLabel = props.i18n.t('in');
+  const allCategories = props.i18n.t('all-categories');
+  const localizedPath = props.value.path.length
+    ? props.value.path.map((value) =>
+        getFieldValueCaption(props.field, value, props.i18n)
+      )
+    : [allCategories];
+  const ariaLabel = props.i18n.t('under', {
+    child: props.i18n.t('facet-value', {
+      count: props.value.count,
+      formattedCount: count,
+      value: props.value.displayValue,
+    }),
+    parent: localizedPath.join(', '),
+  });
+
+  return html`<li data-key=${props.value.displayValue}>
```

Comment:
is `data-key` really what we want? Shouldn't we use the `$keyed` Lit directive instead?